src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/stores/FieldStore.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import type {ComponentType} from 'react';
+
+class FieldStore {
```

Comment:
Refactor to use a common base class

Reply:
I also don't like the `Store` suffix. Would suggest `Registry`

Reply:
Then it would also have to be `RouteRegistry` and `ViewRegistry`. And I still think that it is not a good idea, because the two concepts are too similar, and we would end up with a random mix of both. E.g. would it have to be `ResourceMetadataStore` or `ResourceMetadataRegistry`? That's one I am really not sure about...

But I can change that for all three of them in a separate PR if you insist.

Reply:
+1